docs/en/rst/installing/mysql.rst

```diff
@@ -3,8 +3,12 @@
 MySQL
 #####
 
-You need MySQL version 5.0.15 or higher, but not MySQL 8 or higher.
-MariaDB should work just fine instead if you can't get a hold of older MySQL.
+We recommend MariaDB, since it's compatible with MySQL, and it's more
+compatible with Bugzilla. If you instead want to use MySQL, you will
+need MySQL version 5.0.15 or higher, but not MySQL 8 or higher.
+
+A large part of this page mentions MySQL, but it doesn't matter if you
+use MariaDB or MySQL, the commands are gonna be the same.
 
```

Comment:
same as the 5.0 PR.  "going to"

Reply:
still waiting on this one

Reply:
It looks like it's fixed, but github didn't label it as outdated for some reason

Reply:
Huh, so it is.  I hit the "View changes" button next to the thing saying you'd resolved it and the diff it showed me still had "gonna" in it.  If I look at the overall files changed, it's indeed gone.  Weird.

Reply:
Yeah, for some reason it only considers 3 commits in that view, you can see it in the filters at the top